textx/model.py

```diff
@@ -144,10 +144,11 @@ class ObjCrossRef(object):
         position(int): A position in the input string of this cross-ref.
     """
 
-    def __init__(self, obj_name, cls, position):
+    def __init__(self, obj_name, cls, position, local_scope_provider):
```

Comment:
`local_scope_provider` needs to be added to the docstring above.

Reply:
docstring corrected.